CastIronChat/twisted-metal
# Fix controller support on mac

It might be this simple: stop using pyglet's `ControllerManager`, and use `pyglet.input.get_controllers()` instead.

This needs @13ruce1337 to test and confirm.

## Testing steps

1. Checkout the branch https://github.com/CastIronChat/twisted-metal/pull/78
2. Upgrade your version of pyglet to the latest: `pip install --upgrade pyglet` or using whatever pycharm stuff you've got
3. Run the code with a controller, see if it works

If you get an error, copy-paste the full error text you see into discord and/or the github PR or Trello card

Share the exact versions of arcade and pyglet you're using.  `pip list` should dump a list of all python libs, probably the easiest way to do it
